templates/interactive/h5p.blade.php

```diff
@@ -0,0 +1,6 @@
+<div class="textbox interactive-content">
+	<span class="interactive-content__icon"></span>
+	<p>{{ __( 'An interactive H5P element has been excluded from this version of the text. You can view it online here:', 'pressbooks' ) }}
+		<a href="{{ $url.$id }}" title="{{ $title }}">{{ $url }}</a>
```

Comment:
I changed exactly to that @SteelWagstaff but in local env it was required to clear cache because the blade templates were cached that's why you couldn't see the change.

If you want to try it locally you should:

1. Find your book (site id)
2. Locate the (site id) / pressbooks/cache and delete all the cache files there.
3. Try to export the book again, you should see the full url